src/PageTypes/BlocksPage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace SilverStripe\StartupThemeComponents\PageTypes;
+
+use Page;
+use SilverStripe\Forms\CheckboxField;
+
+class BlocksPage extends Page
+{
+    private static string $table_name = 'BlocksPage';
+
+    private static array $db = [
+        'ShowHero' => 'Boolean',
+    ];
+
+    private static array $defaults = [
+        'ShowHero' => true,
+    ];
+
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+
+        $fields->insertAfter(
+            'MenuTitle',
+            $metaTitle = CheckboxField::create(
```

Comment:
This shouldnt be a named variable becuase we aren't using it anywhere else
```suggestion
            CheckboxField::create(
```